invenio_rdm_records/requests/access/requests.py

```diff
@@ -59,6 +124,14 @@ def execute(self, identity, uow):
                 GuestAccessRequestSubmitNotificationBuilder.build(request=self.request)
             )
         )
+        uow.register(
+            NotificationOp(
+                GuestAccessRequestSubmittedNotificationBuilder.build(
+                    request=self.request
+                )
+            )
+        )
+        # TODO: add  notification for the verification
```

Comment:
is TODO needed, is there still a hardcoded email ?

Reply:
Thanks, simply a leftover, removed!
(the verification email is the submitted template added above)